Source/Resources/Content/MSGraph/DeviceManagement/GroupPolicyConfigurations/OS - System Sleep Timeouts.json

```diff
@@ -45,5 +45,5 @@
       ]
     }
   ],
-  "description": "Sets the system sleep timeout for devices to never when plugged in and to 20 minutes when on battery."
+  "description": "What is it: Sets the system sleep timeout for devices to never when plugged in and to 20 minutes when on battery."
```

Comment:
**What does this do?**
Sets the system sleep timeout for devices to never when plugged in and to 20 minutes when on battery.

**Why should you use this?**
Windows default behavior is to sleep even when plugged in, preventing users from being able to connect remotely.  @metamenos do you think we should remove the battery setting of this config?

**What is the end user impact?**
Users' machine will not fall asleep when connected to power. 

**Learn more**
NA